docs/azureai/azureai-openai-integration.md

```diff
@@ -1,138 +1,384 @@
 ---
 title: .NET Aspire Azure OpenAI integration (Preview)
 description: Learn how to use the .NET Aspire Azure OpenAI integration.
-ms.topic: how-to
-ms.date: 02/14/2025
+ms.date: 03/06/2025
 ---
 
 # .NET Aspire Azure OpenAI integration (Preview)
 
-In this article, you learn how to use the .NET Aspire Azure OpenAI client. The `Aspire.Azure.AI.OpenAI` library is used to register an `OpenAIClient` in the dependency injection (DI) container for consuming Azure OpenAI or OpenAI functionality. It enables corresponding logging and telemetry.
+[!INCLUDE [includes-hosting-and-client](../includes/includes-hosting-and-client.md)]
 
-For more information on using the `OpenAIClient`, see [Quickstart: Get started generating text using Azure OpenAI Service](/azure/ai-services/openai/quickstart?tabs=command-line%2Cpython&pivots=programming-language-csharp).
+[Azure OpenAI Service](https://azure.microsoft.com/products/ai-services/openai-service) provides access to OpenAI's powerful language models including GPT-4, GPT-3.5-Turbo, and Embeddings model series with the security and enterprise promise of Azure. The .NET Aspire Azure OpenAI integration enables you to connect to Azure OpenAI Service or OpenAI's API from your .NET applications.
```

Comment:
I'd recommend not citing specific model names unless you're in a position to update this every month. What's written here already looks pretty outdated (3.5 Turbo, absence of mention of any o-series models).